packages/cfd/src/Containers/mt5-compare-table-content.tsx

```diff
@@ -20,10 +20,14 @@ import {
     getPreappstoreEuDemoContent,
     getEuFooterButtons,
 } from '../Constants/cfd_compare_account_content';
-import { GetSettings, GetAccountSettingsResponse } from '@deriv/api-types';
+import { GetSettings, GetAccountSettingsResponse, DetailsOfEachMT5Loginid } from '@deriv/api-types';
 import { observer, useStore } from '@deriv/stores';
 import { useCfdStore } from '../Stores/Modules/CFD/Helpers/useCfdStores';
 
+type TTypeOfAccount = {
```

Comment:
@hamza-deriv can't we use `TAccountType`

Reply:
Nope i that type has optional and '' value which conflicts with the types of variable used. Let me check again